[
  {
    "component": "header",
    "variant": 1,
    "value": "getting-the-traceback-as-a-string_key1"
  },
  {
    "component": "text",
    "value": "getting-the-traceback-as-a-string_key2"
  },
  {
    "component": "text",
    "value": "getting-the-traceback-as-a-string_key3"
  },
  {
    "component": "text",
    "value": "getting-the-traceback-as-a-string_key4"
  },
  {
    "component": "text",
    "value": "getting-the-traceback-as-a-string_key5"
  },
  {
    "component": "text",
    "value": "getting-the-traceback-as-a-string_key6"
  },
  {
    "component": "text",
    "value": "getting-the-traceback-as-a-string_key7"
  },
  {
    "component": "text",
    "value": "getting-the-traceback-as-a-string_key8"
  },
  {
    "component": "code",
    "type": "python",
    "title": "",
    "value": ">>> import traceback<br>>>> try:<br>         raise Exception('This is the error message.')<br>except:<br>         errorFile = open('errorInfo.txt', 'w')<br>         errorFile.write(traceback.format_exc())<br>         errorFile.close()<br>         print('The traceback info was written to errorInfo.txt.')"
  },
  {
    "component": "text",
    "value": "getting-the-traceback-as-a-string_key9"
  },
  {
    "component": "text",
    "value": "getting-the-traceback-as-a-string_key10"
  }
]